client/components/LearnerViewDirective.js

```diff
@@ -296,9 +296,9 @@ tie.directive('learnerView', [function() {
 
 
         $scope.questionIds = questionSet.getQuestionIds();
-        $scope.unlockedQuestions = [];
+        $scope.completedQuestions = [];
         for (var i = 0; i < $scope.questionIds.length; i++) {
-          $scope.unlockedQuestions.push(false);
+          $scope.completedQuestions.push(false);
```

Comment:
Yep, that's right. I'm not sure what's confusing here, though (or what comment you would like to add)? I mean, it's kinda true by default that when you first open the application, you haven't completed any of the questions...